File: LQDCoder/NENXAU/NENXAU.pas
var     st1,st2,st3: ansistring;
        i,j,dem,p: longint;
begin
    readln(st1);
    i:=1;
        repeat
            dem:=0;
            for j := i to length(st1) do
                if (st1[j] = st1[j+1]) then inc(dem)
                else 
                    begin
                        p:=j+1;
                        break;
                    end;
            if (dem >= 1) then 
                begin
                    str(dem+1,st3);
                    st2:=st2+st3+st1[i]
                end
            else 
                for j := i to i+dem do st2:=st2+st1[j];
            i:=p;
        until (i > length(st1));
    writeln(st2);
end.
    

File: LQDCoder/NENXAU/NENXAU.cpp
#include <iostream>
#include <string.h>

using namespace std;

string st,st2,st3;
int dem,i,j;
char ch;

int main()
{
	getline(cin,st);
	for (i=0;i<st.length();i++)
	{
		ch=st[i];
		dem=0;
		for (j=i;j<st.length();j++)
		{
			if (st[i] == st[j]) dem++;
			else break;
		}
		if (dem > 1) st2=st2+to_string(dem)+ch;
		else st2=st2+ch;
		i=j-1;
	}
	cout << st2;
}